### PR TITLE
Hm 2

### PR DESCRIPTION
**Pull request by ewgenij87snwork**

### CODE REVIEW

File: src/app/posts/posts.module.ts [hunk 1]
```diff
@@ -2,10 +2,13 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { PostsComponent } from './posts.component';
 import { PostCreateModule } from './post-create/post-create.module';
+import { SharedModule } from '../modules/shared/shared/shared.module';
+import { MatDividerModule } from '@angular/material/divider';
```
Comment (MMaksymivUa) [inline src/app/posts/posts.module.ts:6]: move to shared

File: src/app/posts/post-create/post-create.module.ts [hunk 2]
```diff
@@ -3,10 +3,22 @@ import { CommonModule } from '@angular/common';
...
     declarations: [PostCreateComponent],
-    imports: [CommonModule, FormInputModule, ReactiveFormsModule],
+    imports: [
+        CommonModule,
+        FormInputModule,
+        ReactiveFormsModule,
+        SharedModule,
+        MatFormFieldModule,
+        MatInputModule,
+        MatButtonModule,
```
Comment (MMaksymivUa) [inline src/app/posts/post-create/post-create.module.ts:20]: shared module

File: src/app/posts/post-create/post-create.component.html [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
+    <app-form-input
+      [value]="getControl('subtitle').value"
+      [label]="'Subtitle:'"
+      [inputFormControl]="getControl('subtitle')"
+      (pressedEnter)="onSubmit()"
+    ></app-form-input>
+    <app-form-input
+      [value]="getControl('text').value"
+      [label]="'Text:'"
+      [type]="'textarea'"
```
Comment (MMaksymivUa) [inline src/app/posts/post-create/post-create.component.html:19]:
```
      label="Text"
      type="textarea"
```
You can pass string literals in that approach

File: src/app/app.module.ts [hunk 4]
```diff
@@ -5,10 +5,22 @@ import { AppRoutingModule } from './app-routing.module';
...
     declarations: [AppComponent, HeaderComponent],
-    imports: [BrowserModule, AppRoutingModule, PostsModule],
+    imports: [
+        BrowserModule,
+        AppRoutingModule,
+        PostsModule,
+        BrowserAnimationsModule,
+        MatToolbarModule,
+        MatButtonModule,
+        MatIconModule,
```
Comment (MMaksymivUa) [inline src/app/app.module.ts:22]: create one module for material components

**Comment by MMaksymivUa** [comment]
add screenshot. Good job!

**Comment by ewgenij87snwork** [comment]
page__create post

Fixed.
Thank you!

**Pull request merged**